soh/include/message_data_static.h

```diff
@@ -3,32 +3,12 @@
 
 #include "global.h"
 #include "message_data_fmt.h"
+#include "message_data_textbox_types.h"
```

Comment:
I saw this conversation happen, great idea here since this stuff is so easy to extract! Not really part of the review, just glad to see improvements.

Reply:
It was basically neccesary to include `CustomMessageManager.h` in `randomizer.h`. I ran into very similar issues as what you're running into for the colored models stuff you were working on earlier. Vector enums being defined multiple times etc.

It does clean it up a little though, so I'm happy eiher way :)